content/design-systems-for-developers/react/en/distribute.md

```diff
@@ -92,7 +92,7 @@ dist
 
 Finally, let’s make a couple of changes to `package.json` to ensure consumers of the package get all the information we need. The easiest way to do that is to run `yarn init` -- a command that initializes the package for publication:
 
-```yarn init
+```bash
```

Comment:
I am wondering if this was meant to be:

```bash
yarn init

yarn init v1.16.0
...
```

Reply:
it could be..i thought of it aswell, read it a couple of times and thought of that possibility that the case was to add the bash and then the full output of running yarn. If need be i can adjust the snippet. Feel free to provide feedback

Reply:
@tmeasday thoughts?

Reply:
Yeah I think you have it right.

Reply:
i'm going to make the change and push it tomorrow, as it's almost 3 am where i'm at. Been reviewing the translation and it's almost proofread.